blocks/lists.ts

```diff
@@ -352,15 +352,14 @@ blocks['lists_getIndex'] = {
     const modeMenu = fieldRegistry.fromJson({
       type: 'field_dropdown',
       options: MODE,
-    });
+    }) as FieldDropdown;
```

Comment:
Here (and below) you use `as FieldDropdown` but above you use a conditional to check for defined-ness. Can you make these consistent?